sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java

```diff
@@ -764,8 +765,10 @@ private void exportEvents(
                     // TODO (trask) map OpenTelemetry exception to Application Insights exception better
                     String stacktrace = event.getAttributes().get(SemanticAttributes.EXCEPTION_STACKTRACE);
                     if (stacktrace != null && !shouldSuppress.test(span, event)) {
-                        consumer.accept(
-                            createExceptionTelemetryItem(stacktrace, span, operationName, itemCount));
+                        Boolean exceptionAlreadyLogged = event.getAttributes().get(AiSemanticAttributes.EXCEPTION_LOGGED);
+                        if (exceptionAlreadyLogged == null || !exceptionAlreadyLogged) {
```

Comment:
I'm thinking `EXCEPTION_LOGGED` should be a String, so we can compare the stacktrace and only suppress if it's the same?

Reply:
put the whole stacktrace into this attribute? not sure i understand your comment.